examples/cluster_with_existing_credentials/main.tf

```diff
@@ -25,6 +25,33 @@ resource "castai_cluster" "example_cluster" {
       shape = "medium"
     }
   }
+
+  autoscaler_policies {
+    enabled = true
```

Comment:
I suggest making declaration autoscaler_policies {} mean that enabled = true (you can explicitly set enabled = false). As declaration 
```
autoscaler_policies {
      cluster_limits {
        cpu {
          max_cores = 20
          min_cores = 2
        }
    }
}
```
tells me these are declared and desired rules. 
Everything becomes cleaner:

```
autoscaler_policies {

    cluster_limits {
      cpu {
        max_cores = 20
        min_cores = 2
      }
    }

    node_downscaler {
      empty_nodes {
        enabled = false
      }
    }
    spot_instances {
      clouds = ["gcp","aws"]
      enabled = false
    }

    unschedulable_pods {
      enabled = false
      headroom {
        cpu_percentage = 10
        memory_percentage = 10
      }
    }
  }
}
```

Reply:
@ernestas2k I agree and support your suggestion, but @zilvinasu and Co please share your strong opinion.